.github/workflows/infra_plan.yaml

```diff
@@ -9,11 +9,10 @@ on:
         description: Azure region where the resources will be deployed.
         type: string
         required: true
-      tf_configuration_path:
-        description:
+      base_path:
+        description: The base path on which the script will look for Terraform projects
         type: string
         required: false
```

Comment:
#
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏You either have default or mark it required (the script would fail otherwise)

Reply:
Fixed